notebooks/working/5.0-mab-simulation-causal-graph.md

```diff
@@ -961,6 +961,115 @@ def FitAlternativeRegression(regressions, reg_types, data):
     return regression_results
 ```
 
+## Regressions for each causal graph 
+
+
+### Drive Alone 
+
+```python
+V_Drive_Alone.draw()
+```
+
+```python
+Drive_Alone_Df = bike_data_long[bike_data_long['mode_id']==1]
+Drive_Alone_Df.reset_index(drop=True,inplace=True)
+Drive_Alone_Reg = FitAlternativeRegression(regressions={1:('total_travel_distance','total_travel_cost'),
```

Comment:
Small nitpicks around line length for readability.

Also, having the name of the variable by itself for a multi-line variable definition allows you to change the variable name freely while still having easy to understand git-diffs.

```suggestion
Drive_Alone_Reg =\
    FitAlternativeRegression(
        regressions={1:('total_travel_distance','total_travel_cost'),
```

Similar logic applies to putting `FitAlternativeRegression` by itself on a line.

Reply:
Ah! got it! Lots to learn from you brother! Thanks so much!

Reply:
psh, the learning goes both ways! Thanks for working on this during the craziness going on!